DevCycle.SDK.Server.Common/API/DVCRestClientOptions.cs

```diff
@@ -0,0 +1,12 @@
+using RestSharp;
+
+namespace DevCycle.SDK.Server.Common.API
+{
+    public class DvcRestClientOptions : RestClientOptions
```

Comment:
I'd name this `DVCRestClientOptions` - it breaks C#'s recommended convention - but it makes it more package consistent.